benchmarks/fastrnns/bench.py

```diff
@@ -73,11 +74,15 @@ def train_batch(modeldef):
         gc.collect()
 
         fwd_start_event.record()
+        #print(modeldef.forward.graph_for(*modeldef.inputs))
         forward_output = modeldef.forward(*modeldef.inputs)
         fwd_end_event.record()
 
         # XXX: Use if need to print something
-        # print(modeldef.forward.graph_for(*modeldef.inputs))
+        global print_once
```

Comment:
If I remember correctly, this function is called for different rnn_creator, so why a global "print_once"? I am perfectly fine if this flag is local to a particular training, and only once across all the training steps.

Reply:
Oops, this is not supposed to be here :)